torchbenchmark/models/timm_nfnet/__init__.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import timm.models.nfnet
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+from .nfnet import NFNetConfig
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.CLASSIFICATION
+
+    def __init__(self, device=None, jit=False, variant='dm_nfnet_f0', precision='float32'):
+        super().__init__()
+        self.device = device
+        self.jit = jit
+        self.model = timm.create_model(variant, pretrained=False, scriptable=True)
+        self.cfg = NFNetConfig(model = self.model, device = device, precision = precision)
+        self.model.train()
```

Comment:
what's the purpose of calling train() and then eval() here? 

note that the model does need to be set in 'train' or 'eval' mode at the right time (Before calling training or eval functions).  See this api:
https://github.com/pytorch/benchmark/blob/master/torchbenchmark/util/model.py#L47

you probably want to override it.  

its default impl actually would work for you by default, except you did not implement get_module().  That is supposed to be a required API, but for some reason, it is not included in the 'BenchmarkModel' base class.  looks like that was an oversight.  I'd suggest implementing get_module() and then you don't have to override set_mode.

Reply:
I borrowed this part of code from the timm benchmarking code [train](https://github.com/rwightman/pytorch-image-models/blob/master/benchmark.py#L248) and [inference](https://github.com/rwightman/pytorch-image-models/blob/master/benchmark.py#L193). You are right, I should remove this.

However, I am not very clear about the `get_module()` api's semantic. What is it supposed to do? For example, can you explain [the code here](https://github.com/pytorch/benchmark/blob/master/torchbenchmark/models/BERT_pytorch/__init__.py#L101)?

Reply:
Also, I have updated the code to use get_module(), please help check whether it looks correct. Thanks!